tljh/requirements-base.txt

```diff
@@ -5,20 +5,15 @@
 #          the requirements-txt-fixer pre-commit hook that sorted them and made
 #          our integration tests fail.
 #
-# For JupyterHub 1.x SQLAlchemy below 2.0.0
-SQLAlchemy<2.0.0
 # JupyterHub + notebook package are base requirements for user environment
-jupyterhub==1.*
+jupyterhub==3.*
 notebook==6.*
 # Install additional notebook frontends!
 jupyterlab==3.*
 nteract-on-jupyter==2.*
```

Comment:
Last release, 2019, this is deprecated according to: https://github.com/nteract/nteract/issues/5554#issuecomment-852434391

Should we remove it now as well?

Reply:
Removed.